pkg/streamx/streamx_gen_service_test.go

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2024 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package streamx_test
+
+import (
+	"context"
+
+	"github.com/cloudwego/kitex/client"
+	"github.com/cloudwego/kitex/client/streamxclient"
+	"github.com/cloudwego/kitex/client/streamxclient/streamxcallopt"
+	"github.com/cloudwego/kitex/pkg/serviceinfo"
+	"github.com/cloudwego/kitex/pkg/streamx"
+	"github.com/cloudwego/kitex/server"
+	"github.com/cloudwego/kitex/server/streamxserver"
+)
+
+// === gen code ===
+
+// --- Define Service Method handler ---
+var testServiceInfo = &serviceinfo.ServiceInfo{
+	ServiceName:  "kitex.echo.service",
```

Comment:
nil: this ServiceName should be IDL service name instead of psm format

Reply:
@Marina-Sakai done